### PR TITLE
balena additional naming fixes 

### PR DESCRIPTION
**Pull request by zozo123**
- Balena naming fixing
- Restartmanager unit test fixing

### CODE REVIEW

File: integration-cli/daemon/daemon.go [hunk 1]
```diff
@@ -217,7 +217,7 @@ func (d *Daemon) StartWithLogFile(out *os.File, providedArgs ...string) error {
 		return errors.Wrapf(err, "[%s] could not find docker binary in $PATH", d.id)
 	}
 	args := append(d.GlobalFlags,
-		"--containerd", "/var/run/docker/libcontainerd/docker-containerd.sock",
+		"--containerd", "/var/run/balena/libcontainerd/containerd.sock",
```
Comment (dfunckt) [inline integration-cli/daemon/daemon.go]: Can we not diverge from the convention of this socket file being in the format `<name>-containerd.sock`. Is there a reason to?

File: libcontainerd/remote_unix.go [hunk 2]
```diff
@@ -37,8 +37,8 @@ const (
 	containerdShutdownTimeout    = 15 * time.Second
 	containerdBinary             = "balena-containerd"
 	containerdPidFilename        = "balena-containerd.pid"
-	containerdSockFilename       = "balena-containerd.sock"
-	containerdStateDir           = "containerd"
+	containerdSockFilename       = "containerd.sock"
```
Comment (dfunckt) [inline libcontainerd/remote_unix.go]: Same comment about keeping the convention applies here.

File: pkg/ioutils/concat.go [hunk 3]
```diff
@@ -52,20 +52,20 @@ func (self *concatReadSeekCloser) Read(p []byte) (n int, err error) {
 			return 0, err
 		}
 
-		nA, err := io.ReadFull(self.a, p[:min(len(p), int(self.aSize - self.off))])
+		nA, err := io.ReadFull(self.a, p[:min(len(p), int(self.aSize-self.off))])
```
Comment (dfunckt) [inline pkg/ioutils/concat.go]: This is personal preference, but in general I find spaces around infix operators more readable.

File: runconfig/hostconfig_test.go [hunk 4]
```diff
@@ -167,11 +167,11 @@ func TestPidModeTest(t *testing.T) {
...
-		container.RestartPolicy{}:                                         {true, false, false},
-		container.RestartPolicy{Name: "something", MaximumRetryCount: 0}:  {false, false, false},
-		container.RestartPolicy{Name: "no", MaximumRetryCount: 0}:         {true, false, false},
-		container.RestartPolicy{Name: "always", MaximumRetryCount: 0}:     {false, true, false},
-		container.RestartPolicy{Name: "on-failure", MaximumRetryCount: 0}: {false, false, true},
+		{}: {true, false, false},
+		{Name: "something", MaximumRetryCount: 0}:  {false, false, false},
+		{Name: "no", MaximumRetryCount: 0}:         {true, false, false},
+		{Name: "always", MaximumRetryCount: 0}:     {false, true, false},
+		{Name: "on-failure", MaximumRetryCount: 0}: {false, false, true},
```
Comment (dfunckt) [inline runconfig/hostconfig_test.go]: Much of this commit's changes could be PR'd upstream as well?

File: integration-cli/daemon/daemon.go (hunk 1 above)
Comment (petrosagg) [reply integration-cli/daemon/daemon.go]: @zozo123 did you mean `balena-containerd.sock` here?

File: libcontainerd/remote_unix.go [hunk 5]
```diff
@@ -37,8 +37,8 @@ const (
...
+	containerdStateDir           = "balena-containerd"
```
Comment (petrosagg) [inline libcontainerd/remote_unix.go]: Why do we change this since it wasn't prefixed with docker before?

File: api/types/filters/parse_test.go [hunk 6]
```diff
@@ -90,15 +90,15 @@ func TestFromParam(t *testing.T) {
 		`{"key": "value"}`,
 	}
 	valid := map[*Args][]string{
-		&Args{fields: map[string]map[string]bool{"key": {"value": true}}}: {
+		{fields: map[string]map[string]bool{"key": {"value": true}}}: {
```
Comment (petrosagg) [inline api/types/filters/parse_test.go]: How did this run before our work? I don't think we've touched these files

File: pkg/ioutils/concat.go (hunk 3 above)
Comment (petrosagg) [reply pkg/ioutils/concat.go]: @dfunckt go has a language imposed style so we'll go with it. https://blog.golang.org/go-fmt-your-code

File: api/types/filters/parse_test.go (hunk 6 above)
Comment (zozo123) [reply api/types/filters/parse_test.go]: True.

**Comment by zozo123** [comment]
Thanks Petros!

**Pull request merged**